src/components/menu/examples/menu-open-bottom.tsx

```diff
@@ -0,0 +1,26 @@
+import { MenuItem, ListSeparator } from '@limetech/lime-elements';
+import { Component, h } from '@stencil/core';
+
+/**
+ * Opening to the bottom
```

Comment:
Rather than adding several different examples, I think it makes more sense to have a single example where the placement can be updated, similar to the example on https://popper.js.org/. We don't need to have the fancy dots though, I think it's good enough to just have the supported values in a select.

The same can of course also be seen in the composite example, but it's a little more hidden among all the other options there, so I think it's probably useful to have a separate example for just the positioning.

Reply:
Yeah, I think I could whip something up quite quickly 👍

Reply:
⚡